es/src/main/scala/monix/connect/es/package.scala

```diff
@@ -0,0 +1,15 @@
+package monix.connect
+
+import com.sksamuel.elastic4s._
+import monix.eval.Task
+import monix.execution.internal.InternalApi
+
+package object es {
+
+  @InternalApi private[es] implicit val taskFunctor: Functor[Task] = new Functor[Task] {
+    override def map[A, B](fa: Task[A])(f: A => B): Task[B] = fa.map(f)
```

Comment:
maybe we could just refer to the existing instances from elastic4s? https://github.com/sksamuel/elastic4s/blob/master/elastic4s-effect-monix/src/main/scala/com/sksamuel/elastic4s/monix/instances/TaskInstances.scala